src/configure

```diff
@@ -1,11 +1,10 @@
 #! /bin/sh
 # From configure.in Id.
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.71 for hypre 2.27.0.
+# Generated by GNU Autoconf 2.69 for hypre 2.27.0.
```

Comment:
We need to generate this file with autoconf 2.71

Reply:
Thanks @victorapm. I'll fix that